src/chart/RechartsWrapper.tsx

```diff
@@ -96,6 +97,7 @@ export const RechartsWrapper = forwardRef(
       dispatch(externalEventAction({ handler: onTouchStart, reactEvent: e }));
     };
     const myOnTouchMove = (e: React.TouchEvent<HTMLDivElement>) => {
+      dispatch(touchEventAction(e));
```

Comment:
Do we also need touchStart and touchEnd? Or are they behaving like clicks?

Reply:
It's a little flaky. Sometimes it seems to behave fine like a click, sometimes it does not. Let's test after this merge